compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/scene/ComposeContainer.desktop.kt

```diff
@@ -174,25 +196,13 @@ internal class ComposeContainer(
         updateLifecycleState()
         viewModelStore.clear()
 
-        _windowContainer?.removeComponentListener(this)
+        _windowContainer?.removeComponentListener(windowContainerComponentListener)
         mediator.dispose()
         layers.fastForEach(DesktopComposeSceneLayer::close)
     }
 
-    override fun componentResized(e: ComponentEvent?)  {
-        onChangeWindowSize()
-
-        // Sometimes Swing displays interop views in incorrect order after resizing,
-        // so we need to force re-validate it.
-        container.validate()
-        container.repaint()
```

Comment:
Why do you remove this workaround? It's somehow fixed in different place?

Reply:
I put it back.